django_redshift_backend/base.py

```diff
@@ -86,11 +87,11 @@ def convert_uuidfield_value(self, value, expression, connection, context):
             value = uuid.UUID(value)
         return value
 
-    def distinct_sql(self, fields):
-        if fields:
+    def distinct_sql(self, *args):
```

Comment:
Required `fields` and `params` for [`DISTINCT ON`](https://github.com/django/django/blob/2.1.5/django/db/backends/base/operations.py#L164-L171)

Reply:
It seems that Django-2.1 change `distinct_sql` interface (parameters and return value).
https://github.com/django/django/commit/2162f0983de0dfe2178531638ce7ea56f54dd4e7

2.0
https://github.com/django/django/blob/2.0/django/db/backends/postgresql/operations.py#L212-L216

2.1
https://github.com/django/django/blob/2.1/django/db/backends/postgresql/operations.py#L211-L216

So, I think we need new implementation for Django-2.1 such as:
```
if DJANGO_VERSION < 2.1:
    # current implementation
else:
    # new implementation
```

Reply:
@shimizukawa How is `return super().distinct_sql(*args)` ?